src/Illuminate/Foundation/Auth/AuthenticatesUsers.php

```diff
@@ -121,7 +121,9 @@ protected function authenticated(Request $request, $user)
      * Get the failed login response instance.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @return \Illuminate\Http\RedirectResponse
+     * @return \Symfony\Component\HttpFoundation\Response
```

Comment:
this function returns nothing, why did you return response class?

Reply:
For overloading